package.json

```diff
@@ -90,5 +89,8 @@
     "lint:style",
     "lint:js"
   ],
-  "private": true
+  "private": true,
+  "dependencies": {
+    "@ember/edition-utils": "^1.1.1"
```

Comment:
Should be a dev dependency at this level.

Reply:
fixed.